spec/models/vim_performance_state_spec.rb

```diff
@@ -1,5 +1,40 @@
 RSpec.describe VimPerformanceState do
-  describe ".capture_host_sockets" do
+  let(:capture_time) { 5.hours.ago.beginning_of_hour }
+
+  describe ".capture" do
+    it "uses now for Infra" do
+      hardware = FactoryBot.create(:hardware, :cpu_sockets => 2)
+      host = FactoryBot.create(:host, :hardware => hardware)
+      state = VimPerformanceState.capture(host, capture_time)
+
+      expect(state.timestamp).to be_within(1.minute).of(Time.now.beginning_of_hour)
+    end
+
+    it "uses time passed in for containers" do
+      container_image = FactoryBot.create(:container_image)
+      container = FactoryBot.create(:container, :container_image => container_image)
+      state = VimPerformanceState.capture(container_image, capture_time)
+
+      expect(state.timestamp).to be_within(1.minute).of(capture_time)
+    end
+  end
+
+  describe "#capture_assoc_ids" do
+    it "captures runing containers" do
+      container_image = FactoryBot.create(:container_image)
+      # not deleted is captured
+      containers = FactoryBot.create_list(:container, 1, :container_image => container_image)
```

Comment:
Why is this using create_list? (Minor, but I just wasn't sure if you meant to only create one or were planning on more than one)

Reply:
(Similar question for the other create_list calls below)

Reply:
yea, only creating one, (could create more) but using that over then turning this into an array

Reply:
Oh i see - it's to create an Array of 1.